recipes/_install.rb

```diff
@@ -29,16 +29,16 @@ def nsp
 include_recipe 'cerner_splunk::_restart_marker'
 
 # Actions
-# This service definition is used only for ensuring splunk is started during the run
-service 'splunk-start' do
+# This service definition is used for ensuring splunk is started during the run and to stop splunk service
+service 'splunk' do
   service_name service
   action :nothing
-  supports status: true, start: true
+  supports status: true, start: true, stop: true
   notifies :delete, 'file[splunk-marker]', :immediately
```

Comment:
I hadn't considered that before but yes, the service resource renaming is potentially non-passive.  I'm fine reverting most of those changes for the sake of passivity.  

The service resource naming is unfortunate because the splunk-stop resource is going to be largely identical to the splunk-start resource declaration. And when you use it, things will still look redundant since you'll have to call `notifies :stop, 'service[splunk-stop]'`.  Thoughts on reverting the name change but leaving the stop action support here on the splunk-start resource?